docs/getting-started/installation-and-upgrade/installation-references/helm-chart-options.md

```diff
@@ -92,13 +92,24 @@ Set the `auditLog.destination` to `hostPath` to forward logs to volume shared wi
 
 ### Setting Extra Environment Variables
 
-You can set extra environment variables for Rancher server using `extraEnv`. This list uses the same `name` and `value` keys as the container manifest definitions. Remember to quote the values.
+You can set extra environment variables for Rancher server using `extraEnv`. This list is passed to the rancher deployment in its YAML format. It is embedded under "env" for the rancher container. extraEnv can use any of the keys described by the [Define Environment Variables for a Container](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#define-an-environment-variable-for-a-container) Kubernetes docs for setting container environment variables.
+
+An example that uses the `name` and `value` keys:
 
 ```plain
 --set 'extraEnv[0].name=CATTLE_TLS_MIN_VERSION'
 --set 'extraEnv[0].value=1.0'
 ```
 
+If passing sensitive data as the value for an environment variable, it is strongly recommended that a secret reference is used. This will prevent sensitive data from being exposed in helm or the rancher deployment.
```

Comment:
```suggestion
If passing sensitive data as the value for an environment variable, like proxy authentication credentials, for example, it is strongly recommended that a secret reference is used. This will prevent sensitive data from being exposed in Helm or the Rancher deployment.
```